packages/core/src/families/solana/types.ts

```diff
@@ -167,6 +167,7 @@ export type TransactionModel = { commandDescriptor?: CommandDescriptor } & (
 export type SolanaTransaction = TransactionCommon & {
   readonly family: RawSolanaTransaction["family"];
   model: TransactionModel;
+  raw?: string;
```

Comment:
This could give a little more infos on the what's raw
```suggestion
  // base64-encoded serialized transaction
  rawTx?: string;
```
or
```suggestion
  // base64-encoded serialized transaction
  rawTransaction?: string;
```